core/src/eav/memory/mod.rs

```diff
@@ -0,0 +1,212 @@
+use eav::{Attribute, Entity, EntityAttributeValue, EntityAttributeValueStorage, Value};
+use error::HolochainError;
+use std::collections::HashSet;
+
+pub struct MemoryStorageEav {
```

Comment:
How about renaming this to `EavMemoryStorage` ?
that way we can rename the current `MemoryStorage` to `ContentMemoryStorage` and have consistent naming.

Reply:
That would be perfect! I will make those changes asap